db/migrate/20150729170224_add_boxes.rb

```diff
@@ -0,0 +1,9 @@
+class AddBoxes < ActiveRecord::Migration
+  def change
+    create_table :boxes do |t|
+      t.string :month, null: false
+      t.string :completed, default: false
```

Comment:
I prefer the published? Flag so that way we can filter non published boxes and can test how a box will look like from the admin panel.